scripts/rnnlm/make_word_features.py

```diff
@@ -87,14 +86,14 @@ def read_features(features_file):
     with open(features_file, 'r', encoding="utf-8") as f:
         for line in f:
             fields = line.split()
-            assert len(fields) == 2 or len(fields) == 3
+            assert(len(fields) in [2, 3, 4])
 
             feat_id = int(fields[0])
             feat_type = fields[1]
             if feat_type == 'special':
                 feats['special'][fields[2]] = feat_id
             elif feat_type == 'unigram':
-                feats['unigram'] = (feat_id, float(fields[2]))
+                feats['unigram'] = (feat_id, float(fields[2]), float(fields[3]))
```

Comment:
Please also update the documentation for feats['unigram'] at the beginning of this function